xen/drivers/passthrough/Makefile

```diff
@@ -7,3 +7,4 @@ obj-y += iommu.o
 obj-$(CONFIG_X86) += io.o
```

Comment:
It was considered by me and described in iommu integration design doc. I can drop this sentence at all.